source/containers.tex

```diff
@@ -879,6 +887,25 @@
 }
 
 \indexcont{operator<=>}%
+\indexlibrarymember{array}{operator<=>}%
```

Comment:
`\indexcont` almost does this; if we feel we want both orderings for indexing, please fix `\indexcont` instead of near-duplicating its definition.

But, maybe we don't, and all that's missing is `\indexlibrarymemberx` for `basic_string`.